alloc-fmt/Cargo.toml

```diff
@@ -21,7 +21,12 @@ repository = "https://github.com/ezrosent/allocators-rs/tree/master/alloc-fmt"
 
 exclude = ["appveyor.sh", "travis.sh"]
 
+[features]
+default = ["print_backtrace"]
+
+print_backtrace = ["backtrace"]
```

Comment:
Can you rename this to `print-backtrace` (a dash instead of an underscore) to be consistent with features in other crates in this project?